Added arrows to the poster slider

nimradium opened the pull request:
* fonts/slick: 
	* Folder of the slick font (contains the arrows)

* main.js: 
	* added the arrows to the slider
	* made them responsive (remove on screens < 736px)

* main.css:
	* added the font to css
	* added and modified the css for the arrows
	* made them responsive for smaller screens

[Working repository](https://github.com/nimradium/Rann-Neeti-2018)

vsvipul commented:
Can u give screenshots showing what exactly did u change? Arrows were visible before too, right ?

nimradium commented:
> Arrows were visible before too, right ?

@vsvipul no they were disabled because they require a font that wasn't "installed" yet.

I can give you screenshots but I only added the needed font and css, added the arrows and modified the page and css so it looks good on all screen sizes (for example they are disabled on screens smaller than 736 px)

vsvipul commented:
i see now the posters shuffle automatically when scrolled down.. that is a good change.. also for the arrows, i checked that .. good work.. i'll proceed to merge this to master.. :+1:
